tinyxml2.cpp

```diff
@@ -2304,6 +2282,17 @@ void XMLDocument::Parse()
     ParseDeep(p, 0, &_parseCurLineNum );
 }
 
+template<class NodeType, int PoolElementSize>

```

Comment:
It seems safest; as I said, this is a weird corner of c++ compiler functionality.